akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -83,7 +88,7 @@ class Http2ServerDemux extends GraphStage[BidiShape[Http2SubStream, FrameEvent,
 
       override protected def logSource: Class[_] = classOf[Http2ServerDemux]
 
-      val multiplexer = createMultiplexer(frameOut, StreamPrioritizer.first())
+      override val multiplexer = createMultiplexer(frameOut, StreamPrioritizer.first())
```

Comment:
Let's not add `override`s in arbitrary places. `override` was supposed to be used as a clue to the compiler and the reader's eye to show that a method is actually overridden (in contrast to just being implemented). Putting it in arbitrary places means that we actually lose a bit of safety where an implementation is added to the super-class later on.

I know the akka codebase had the rule somewhere that to add `override` everywhere possible. IMO it doesn't help and was never enforced strictly so it is not really useful. If in doubt, let's go with the more concise version.